content/_components/ebay/actions.md

```diff
@@ -13,108 +13,84 @@ ComponentVersion: 2.0.2
 
-* **PRODUCT** - This adds the additionalImages, additionalProductIdentities, aspectGroups, description, gtins, image, and title product fields to the response, which describe the product associated with the item. See Product for more information about these fields.
-* **COMPACT** - This returns only the following fields, which let you quickly check if the availability or price of the item has changed, if the item has been revised by the seller, or if an item's top-rated plus status has changed for items you have stored.
 Restriction: Must be a top-level category
 
 More info about this action could be find [here](https://developer.ebay.com/api-docs/buy/browse/resources/item/methods/getItem).
 
-### Input json schema location
-
-Request JSON schema ./src/main/resources/schemas/json/Item.in.json
-
 ## Get Item Feed
 
 The Item feed file is generated each day. This call lets you download a daily TSV_GZIP (tab separated value gzip) Item feed file of all the items that were listed on a specific day in a specific category.
 
-![Get Item Feed](img/get-item-feed.png)
+{% include img.html max-width="100%" url="img/get-item-feed.png" title="Get Item Feed" %}
 
 Details about the TSV_GZIP file could be found here: [Retrieve a daily Item feed file](https://developer.ebay.com/api-docs/buy/static/api-feed.html)
 
 ### Input fields description (if there is any)
 
-__date__ - The date of the feed file you want. The Item feed files are generated every day and there are always 7 daily feed files available.
-
-Format: yyyyMMdd
+* **date** - The date of the feed file you want. The Item feed files are generated every day and there are always 7 daily feed files available.
 
-Requirement: Must be within 3-10 days in the past
+  * **Format:** `yyyyMMdd`
+  * **Requirement:** Must be within 3-10 days in the past
 
-__categoryId__ - An eBay top-level category ID of the items to be returned in the feed file. The list of eBay category IDs changes over time and category IDs are not the same across all the eBay maketplaces. To get a list of the top-level categories for a markeplace, you can use the Taxonomy API getCategoryTree call. This call retrieves the complete category tree for the marketplace. The top-level categories are identified by the categoryTreeNodeLevel field.
+* **categoryId** - An eBay top-level category ID of the items to be returned in the feed file. The list of eBay category IDs changes over time and category IDs are not the same across all the eBay maketplaces. To get a list of the top-level categories for a markeplace, you can use the Taxonomy API getCategoryTree call. This call retrieves the complete category tree for the marketplace. The top-level categories are identified by the categoryTreeNodeLevel field.
```

Comment:
* **categoryId** - An eBay top-level category ID of the items to be returned in the feed file. The list of eBay category IDs changes over time and category IDs are not the same across all the eBay marketplaces. To get a list of the top-level categories for a marketplace, you can use the Taxonomy API `getCategoryTree` call. This call retrieves the complete category tree for the marketplace. The top-level categories are identified by the category `TreeNodeLevel` field.